ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -427,13 +427,25 @@ objects:
         terraform apply -auto-approve -var 'bootstrap_complete=true' -no-color &
         wait "$!"
 
+        function approve_csrs() {
+          while true; do
+            sleep 5;
+            oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs --no-run-if-empty oc adm certificate approve && break
```

Comment:
Are we looping here because we have seen cases where the CSRs were not available yet? If the CSRs trickle in instead of all coming in at once, then this has the potential to approve some but not all of the CSRs. Should we keep approving CSRs until `wait-for install-complete` succeeds?

Reply:
Yeah, the idea is that by the time bootstrap is destroyed all nodes have created a CSR. Not sure if there is a simple way to run CSR approval loop until `install-complete`

Reply:
Could we not create a file after `wait-for install-complete` and break out of the `approve_csrs` while loop when that file exists?